src/test/resources/domain/GET/global-max-recursions.yml

```diff
@@ -0,0 +1,17 @@
+step_1:
```

Comment:
Needs a bit clarification - if global value for `maxRecursions` is set to `100`, will all of the steps below be run
1. only once as `maxRecursions` has not been defined;
2. always 100 times each as this is the global value?

Reply:
By default all steps will be run once. In case you call out a step recurringly, then maxRecursions limits how many times it can be run. So in this DSL, step2 and step3 will create a recursion because step2 calls step3 and step3 calls step2, which creates recursion. When the global value maxRecursions is 100 and the step2 or step3 reaches that many recursions, the DSL will continue with the next step, which in this case is return_step.

Reply:
Excellent!